dev/com.ibm.ws.microprofile.metrics.2.3.monitor/src/com/ibm/ws/microprofile/metrics/monitor/MonitorMetricsHandler.java

```diff
@@ -57,6 +60,11 @@ protected void activate(ComponentContext context) {
     public void getSharedMetricRegistries(SharedMetricRegistries sharedMetricRegistry) {
         this.sharedMetricRegistry = sharedMetricRegistry;
     }
+	
+    @Reference
+    public void getExecutorService(ExecutorService execServ) {
```

Comment:
This is usually called `setExecutorService()` and there should be a corresponding `unsetExecutorService()`.

Reply:
I don't think that service will be unloaded unless the server is shut down. But added it anyways.